LondonBritishLibrary/orient/BLorient691.xml

```diff
@@ -54,67 +294,111 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">243</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>365</height>
-                                        <width>235</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>14.375</height>
+                                        <width>9.25</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="2" writtenLines="22 26" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is written in a fine  hand, of the <date>15th century</date>. 
-                                    
+                                <desc>The manuscript is written in a fine hand, of the <date>15th century</date>.
+
                                 </desc>
-                                
+
                                 <date notBefore="1400" notAfter="1500" resp="PRS10171Wright"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
-                            
+<list>
+
+  <item xml:id="a1">
+  <locus target="#2v"/>
+  <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+  <placeName ref="INS0101MadhaneAlam"/>.</desc>
+  <q xml:lang="gez">ገድለ፡ ጊዮርጊስ፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+  </item>
+
+  <item xml:id="a2">
+  <locus target="#214v"/>
+  <desc type="Unclear"> መልአ፡ መጽሐፈ፡ እንባእቆም፡ ነቢይ፡ ዐቢይ፡ አመ፡ ፲ወ፰፡ ለጥቅምት፡ በ<date calendar="grace" when="0477">፬፻፸፯: ዓመተ፡ ምሕረት፡</date> በመዋዕለ፡ <persName ref="PRS10215Yagbas">ያግባ፡<sic/> ጽዮን፡</persName> ንጉሥ፡ መፍቀሬ፡
+  እግዚአብሔር፡ ዘተሠርዐ፡ በመዋዕሊሁ፡ ከመ፡ ይትአዘዙ፡ ሕዝበ፡ ኢትዮጵያ፡፡ ከመ፡ ይትዓገሱ፡ ከመ፡ ክርስቲያን፡ በከ...>
+  </desc>
```

Comment:
The incipit should be encoded in q with the seg. In the desc you can maybe summarize that this is a note on completion of the book in a later hand with a unrealistic dating

Reply:
Sorry, I can not see that change.